.circleci/config.yml

```diff
@@ -7,16 +7,16 @@ jobs:
       - run: | 
           TAG=0.1.$CIRCLE_BUILD_NUM
           docker login -u $DOCKER_USER -p $DOCKER_PASS
-          docker build -t ajaytripathy/kubecost-cost-model:$TAG .
-          docker push ajaytripathy/kubecost-cost-model:$TAG
+          docker build -t kubecost/cost-model:$TAG .
```

Comment:
Let's use quay.io/kubecost1/kubecost-cost-model:latest for consistency with the image in the kubernetes/deployment folder

Reply:
updated